src/fuego/run_on_spaces.py

```diff
@@ -472,13 +471,13 @@ def run(
         path_or_fileobj=_start_server_template.format(
             command=command,
             status_checker_args=convert_dict_to_args_str(
-                dict(
-                    this_space_repo_id=space_id,
-                    output_dataset_id=dataset_id,
-                    output_dirs=space_output_dirs,
-                    delete_on_completion=delete_space_on_completion,
-                    downgrade_hardware_on_completion=downgrade_hardware_on_completion,
-                )
+                {
```

Comment:
Don't like this change, is there something we can do about it?

Reply:
Yeah sure, let me check PEP 👍🏻

Reply:
This can certainly be disabled, but there are two benefits:

- Creation is faster
- String keys allow for characters disallowed as Python identifiers, like hyphens